src/components/MainView/index.jsx

```diff
@@ -171,10 +171,23 @@ class MainView extends React.Component {
    * @param {ScenarioObject} scenario
    */
   _renderScenario (scenario) {
+    const {
+      params: { context, scenario: name }
+    } = this.props.routeData
+
+    // Fallbacks if scenario is not found
+    const onAcceptSnapshot = scenario
+      ? () => this._acceptSnapshot(scenario)
+      : () => null
```

Comment:
`() => scenario ? this._acceptSnapshot(scenario) : null`
or even
`() =? scenario && this._acceptSnapshot(scenario)` if false is good enough as a value in this case.